src/Ledger/GovernanceActions.lagda

```diff
@@ -112,7 +183,48 @@ NeedsHash (TriggerHF _)        = GovActionID
 NeedsHash (ChangePParams _ _)  = GovActionID
 NeedsHash (TreasuryWdrl _)     = ⊤
 NeedsHash Info                 = ⊤
+\end{code}
 
+A governance action is an on-chain event that is triggered by a transaction, and there is a deadline after which
```

Comment:
Since this document is only about on-chain things we don't need to mention this. Also, an `event` isn't a properly defined thing, so I'd leave that out as well. Here, a governance action is just some data that can be included in a transaction & which can trigger further rules in the ledger.

Reply:
Okay, thanks for the clarification.  I'll eliminate "on-chain event."